springfox-schema/src/test/groovy/springfox/documentation/schema/property/field/FieldModelPropertyProviderSpec.groovy

```diff
@@ -91,4 +91,30 @@ class FieldModelPropertyProviderSpec extends Specification {
       deSerializationPropNames == ['name']
 
   }
+
+  def "Handles enum collections" () {
+
+    given:
+    def genericNamingStrategy = new DefaultGenericTypeNamingStrategy()
+    Class typeToTest = collectionEnumType()
+    def typeResolver = new TypeResolver()
+    ResolvedType resolvedType = typeResolver.resolve(typeToTest)
+    ObjectMapper mapper = objectMapperThatUsesFields()
+    def namingStrategy = new ObjectMapperBeanPropertyNamingStrategy()
+    namingStrategy.onApplicationEvent(new ObjectMapperConfigured(this, mapper))
+    def propertyProvider = new FieldModelPropertyProvider(new FieldProvider(typeResolver),
+            namingStrategy, defaultSchemaPlugins(), Mock(TypeNameExtractor))
+    propertyProvider.objectMapper = mapper
+    def serializationPropNames = propertyProvider.propertiesFor(resolvedType, returnValue(resolvedType, DocumentationType.SWAGGER_12, alternateTypeProvider(), genericNamingStrategy)
+    )
+            .collect({it.name})
+    def deSerializationPropNames = propertyProvider.propertiesFor(resolvedType, inputParam(resolvedType, DocumentationType.SWAGGER_12, alternateTypeProvider(), genericNamingStrategy)
+    )
+            .collect({it.name})
+
+    expect:
+    serializationPropNames == ['exampleEnums']
+    deSerializationPropNames == ['exampleEnums']
```

Comment:
May be just verify that the properties itself rather than just the names, since this is new behavior

```groovy
def deSerializationProps = propertyProvider.propertiesFor(resolvedType, inputParam(resolvedType, DocumentationType.SWAGGER_12, alternateTypeProvider(), genericNamingStrategy))
expect:
     def prop = deSerializationProps.first()
     prop.name == "exampleEnums"
     .... //assert has allowable values etc.
    // same for serialization propererties
```